sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorLiveTestBase.cs

```diff
@@ -103,6 +103,6 @@ public AnomalyDetectionConfiguration PopulateMetricAnomalyDetectionConfiguration
         internal DataFeedGranularity _dailyGranularity;
         internal DataFeedSchema _dataFeedSchema;
         internal DataFeedIngestionSettings _dataFeedIngestionSettings;
-        internal DataFeedOptions _dataFeedOptions;
+        internal string _dataFeedDescription;
```

Comment:
`_dataFeedOptions` was an internal field we were using in our tests to configure data feeds. `Description` was the only property that we were actually using, so I replaced it with a `_dataFeedDescription` string.